html/includes/print-menubar.php

```diff
@@ -187,11 +187,11 @@
                 echo('
                     <li role="presentation" class="divider"></li>
                     <li class="dropdown-submenu">
-                    <a href="#"><i class="fa fa-map-marker fa-fw fa-lg" aria-hidden="true"></i> Geo Locations</a>
+                    <a href="locations"><i class="fa fa-map-marker fa-fw fa-lg" aria-hidden="true"></i> Geo Locations</a>
```

Comment:
This stops you being able to click on a mobile to expand that menu. When you do it just takes you to the locations url.